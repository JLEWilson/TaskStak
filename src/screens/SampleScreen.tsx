import { StatusBar } from "expo-status-bar"
import { StyleSheet, Text, View } from "react-native"

export default function SampleScreen() {
  return (
    <View style={styles.container}>
      <Text>This is to test react-test-renderer</Text>
    </View>
  )
}

const styles = StyleSheet.create({
  container: {
    backgroundColor: "red",
  },
})
