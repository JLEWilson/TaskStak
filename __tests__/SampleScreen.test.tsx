import React from "react"
import renderer from "react-test-renderer"
import SampleScreen from "../src/screens/SampleScreen"

test("renders correctly", () => {
  const tree = renderer.create(<SampleScreen />).toJSON()
  expect(tree).toMatchSnapshot()
})
