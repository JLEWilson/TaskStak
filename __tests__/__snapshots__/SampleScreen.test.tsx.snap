// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "red",
    }
  }
>
  <Text>
    This is to test react-test-renderer
  </Text>
</View>
`;
